src/60_Samples_%26_Templates/News_Article.html

```diff
@@ -83,17 +83,17 @@
 
           --box-shadow-1: 0 1px 1px 0 rgba(0,0,0,.14), 0 1px 1px -1px rgba(0,0,0,.14), 0 1px 5px 0 rgba(0,0,0,.12);
           }
-          figure {
+          .figure {
             margin: var(--space-2) 0;
             padding: 0;
           }
-          figure>figcaption{
+          .figure>figcaption{
             padding: var(--space-1) var(--space-2);
           }
           .amp-ad-container {
             display: flex;
           }
-          amp-ad {
+          .amp-ad {
```

Comment:
suggestion: use `amp-ad-container amp-ad` to avoid declaring another class

Reply:
done